src/Stratis.Bitcoin.Features.Miner/PosBlockAssembler.cs

```diff
@@ -41,12 +41,16 @@ public override BlockTemplate CreateNewBlock(Script scriptPubKeyIn, bool fMineWi
 
             base.CreateNewBlock(scriptPubKeyIn, fMineWitnessTx);
 
-            this.coinbase.Outputs[0].ScriptPubKey = new Script();
-            this.coinbase.Outputs[0].Value = Money.Zero;
-
             PosConsensusValidator posValidator = this.consensusLoop.Validator as PosConsensusValidator;
             Guard.NotNull(posValidator, nameof(posValidator));
 
+            // Without this the coinbase transaction has a zero value
+            this.coinbase.Outputs[0].Value = this.fees + posValidator.GetProofOfWorkReward(this.height);
```

Comment:
this seems like an incompatible change, haven't tested though, but what makes you think it is a good idea?

Reply:
Considering that the functionality previously did not work at all, incompatibility is a given.

Without those lines, the Stratis POW blocks have a single transaction inside them with 0 value. This is not correct. There should be 4 Stratis coins generated per block in the POW phase.

Reply:
>There should be 4 Stratis coins generated per block in the POW phase.

50. 
See Network.cs : 799